llama-rs/src/lib.rs

```diff
@@ -297,6 +311,28 @@ macro_rules! mulf {
     };
 }
 
+trait FromLeBytes {
```

Comment:
I couldn't really figure out a cleaner way to do a trait bound for a bunch of different 32bit long numerical primitives.